specs/olm.spec

```diff
@@ -28,7 +28,12 @@ Steps:
     | S.NO | resource_dir                   |
     |------|--------------------------------|
     | 1    | testdata/hub/tektonhub.yaml    |
-
+  * Switch to project "openshift-pipelines"
+  * Apply 
+    | S.NO | resource_dir                          |
+    |------|---------------------------------------|
+    | 1    | testdata/resolvers/git-secret.yaml    |
+  * Edit git-resolver-config in the TektonConfig
```

Comment:
I would go with something more descriptive, maybe `Configure GitHub token for git resolver in TektonConfig`